Source/EasyTipView.swift

```diff
@@ -408,7 +409,9 @@ open class EasyTipView: UIView {
     // MARK:- Callbacks -
     
     func handleTap() {
-        dismiss()
+        if preferences.dismissOnTap {
```

Comment:
I would prefer it done differently:

On line 110, where the `UITapGestureRecognizer` gets created, I would set the tip view as it's delegate and then in the `gestureRecognizerShouldBegin(_:)` function, I would `return preferences.dismissOnTap`.

That's a cleaner approach.